src/core_ocean/shared/mpas_ocn_gm.F

```diff
@@ -516,7 +529,6 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, diagnosticsPool, &
       deallocate(dTdzTop)
       deallocate(dSdzTop)
       deallocate(k33Norm)
-      deallocate(slopeTaper)
```

Comment:
no `deallocate(slopeTaperEdge)`?

Reply:
I put slopeTaperEdge in registry now in case we want to visualize it at some point.  This is why there is no deallocate here.